add Dutch translation definition 🇳🇱 (#55)
ContributorCovenant/ethicalsource: content/definition.nl.md

+++
title = "De Ethische Source Definitie (ESD)"
template = "definition"
heading="De Ethische Source Definitie"
+++

Software kan als **Ethische Open Source** (EOS) beschouwd worden als en wanneer het aan *alle* criteria voldoet:

1. **Het komt de maatschappij ten goede.** De licentie waaronder de software wordt vrijgegeven mag modificatie, afgeleide werken en het linken of compileren met andere software (tenzij onder een auteursplichtbepaling) in de algemene geest van open source-distributie niet verbieden.

> 2. **Het is openbaar.** De source code moet publiek beschikbaar zijn, ontwikkeld en onderhouden worden, en openbare bijdragen verwelkomen (afhankelijk van beoordeling en goedkeuring door de beheerders van het softwareproject).

Reviewer: ```suggestion
2. **Het is openbaar.** De broncode moet publiek beschikbaar zijn, ontwikkeld en onderhouden worden, en openbare bijdragen verwelkomen (afhankelijk van beoordeling en goedkeuring door de beheerders van het softwareproject).
```
maybe?

Author: I might use English terminology disproportionately.

> 3. **De community is gastvrij en rechtvaardig.** De community van het software project - van maintainers en bijdragers - moet worden gestuurd door een uitgebreide gedragscode die consistent en eerlijk wordt gehandhaafd.

Reviewer: ```suggestion
3. **De community is gastvrij en rechtvaardig.** De community van het softwareproject - van maintainers en bijdragers - moet worden gestuurd door een uitgebreide gedragscode die consistent en eerlijk wordt gehandhaafd.
```

4. **Toegankelijkheid staat voorop.** Als de software een gebruikersinterface heeft, moet deze worden ontworpen met het oog op toegankelijkheid, zodat alle functionaliteit beschikbaar is voor alle gebruikers, inclusief gebruikers die op ondersteunende apparaten vertrouwen.
5. **Het geeft prioriteit aan gebruikersveiligheid.** De software moet zijn ontworpen met functies en beveiligingen die het risico van misbruik of schade voor anderen door het gebruik van de software minimaliseren.

> 6. **Het beschermt de privacy van gebruikers.** If the software collects end-user data, the entity running the software must, upon request,  delete or provide to the end-user any and all data generated and stored from the end-user’s usage of the software. All such data provided to the end-user must be in either a human-readable or widely interoperable open format. 

Reviewer: Missed a spot

Als de software gegevens van eindgebruikers verzamelt, moet de entiteit die de software uitvoert op verzoek alle gegevens die zijn gegenereerd en opgeslagen door het gebruik van de software door de eindgebruiker verwijderen of aan de eindgebruiker verstrekken. Al deze gegevens die aan de eindgebruiker worden verstrekt, moeten in een leesbaar of breed interoperabel open formaat zijn.

> 7. **Het moedigt een eerlijke vergoeding aan.** De beheerders van het software project kunnen, naar eigen goeddunken, een vergoeding vragen in de vorm van code bijdragen, financiële compensatie of andere vormen van vrijwillige ondersteuning van organisaties die commerciële waarde aan de software ontlenen.

Reviewer: ```suggestion
7. **Het moedigt een eerlijke vergoeding aan.** De beheerders van het software project kunnen, naar eigen goeddunken, een vergoeding vragen in de vorm van codebijdragen, financiële compensatie of andere vormen van vrijwillige ondersteuning van organisaties die commerciële waarde aan de software ontlenen.
```

> Ethische Open Source projecten worden aangemoedigd om de badge "Open|Ethical" te gebruiken in de documentatie van hun project: 

Reviewer: ```suggestion
Ethische Open Source-projecten worden aangemoedigd om de badge "Open|Ethical" te gebruiken in de documentatie van hun project: 
```

* Het wordt gedistribueerd onder de ["Anti 996" licentie](https://996.icu/), of een vergelijkbare licentie die de herverdeling van de software in gewijzigde, afgeleide of verzamelde werkvorm aan een persoon of organisatie die inbreuk pleegt op de [International Labour Standards Conventions and / or Recommendations](https://www.ilo.org/global/standards/introduction-to-international-labour-standards/conventions-and-recommendations/lang--en/index.htm), uitgegeven door de Internationale Arbeidsorganisatie, verbiedt.
* Het wordt gedistribueerd onder een ["Atmosphere" licentie](https://www.open-austin.org/atmosphere-license/), of een vergelijkbare licentie die de herverdeling van de software in gewijzigde, afgeleide of verzamelde werkvorm aan een persoon of organisatie die materieel en rechtstreeks investeert in activiteiten of faciliteiten die een aanzienlijke negatieve impact hebben op de wereldwijde klimaatrespons, zoals [gedocumenteerd door het secretariaat van de Verenigde Naties voor klimaatverandering](https://unfccc.int/resource/climateaction2020/media/1308/unfccc_spm_2018.pdf), verbiedt.

> \* *De term "open source" verwijst in deze context naar de brede ervaring van openbare en samenwerkende softwareontwikkeling als bijdrage aan de collectieve softwaregemeenschap. ELOS software licenties voldoen al dan niet aan de [conventionele criteria van het Open Source Initiative](https://opensource.org/osd-annotated) voor Open Source software licenties.*

Reviewer: ```suggestion
\* *De term "open source" verwijst in deze context naar de brede ervaring van openbare en samenwerkende softwareontwikkeling als bijdrage aan de collectieve softwaregemeenschap. ELOS softwarelicenties voldoen al dan niet aan de [conventionele criteria van het Open Source Initiative](https://opensource.org/osd-annotated) voor Open Source softwarelicenties.*
```

Ethisch gelicentieerde Open Source projecten worden aangemoedigd om de badge 'Licensed Ethically' te gebruiken in de documentatie van hun project: 

![https://img.shields.io/badge/licensed-ethically-%234baaaa](https://img.shields.io/badge/licensed-ethically-%234baaaa) 

[https://img.shields.io/badge/licensed-ethically-%234baaaa](https://img.shields.io/badge/licensed-ethically-%234baaaa)

## Naamsvermelding en bijdragen

> Het handhaven van deze definitie is een samenwerking van de [Ethical Source Working Group](/apply). Pull requests en issues in onze [source code repository](https://github.com/ethicalSource/ethicalsource.dev) worden aangemoedigd. Speciale dank gaat uit naar [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), en [Madalyn Parker](https://twitter.com/madalynrose).

Reviewer: ```suggestion
Het handhaven van deze definitie is een samenwerking van de [Ethical Source Working Group](/apply). Pull requests en issues in onze [source code repository](https://github.com/ethicalSource/ethicalsource.dev) worden aangemoedigd. Speciale dank gaat uit naar [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com) en [Madalyn Parker](https://twitter.com/madalynrose).
```
Oxford comma doesn't apply in Dutch, idk why.
